lib/blacklight/models/file.rb

```diff
@@ -11,11 +13,17 @@ class BlacklightFile
     def initialize(zip_entry)
       path = zip_entry.name
       id = File.basename(path)
+      xid = id[/__(xid-[0-9]+_[0-9]+)/, 1]
```

Comment:
@conradbeach Yes, that is super slick. I didn't know you could do that.
@bfcoder The same regex is used on the next line to remove the xid from the name